sdk/purview/Azure.Analytics.Purview.Workflows/CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Release History
+
+## 1.0.0-beta.2 (Unreleased)
```

Comment:
This needs to match the version in the `.csproj` file, which still reads `1.0.0-beta.2`

Reply:
Fixed, thanks